tools/wagmi-example/README.md

```diff
@@ -0,0 +1,21 @@
+# Wagmi Hedera Example
+
+## How to run example?
+1. Install dependencies `pnpm install`
+2. Run project `pnpm dev`
+
+## Example
+Example of using Wagmi with Hedera covers
+1. Sign in with wallet
+2. Check account balance
+3. Deploy contract from bytecode
+4. Read from contract
+5. Write to contract
+
+## Generate contract abi, bin
+if you want compile contract at your own there is a known issue with the two latest version of the solc compiler. You need to use version `solc <= 0.8.24` because the recent version of the Hedera doesn't support all opcodes from the latest release.
```

Comment:
> If you want **TO** compile the contract **ON** your own, be aware of a known issue with the two latest versions of the Solc compiler. You should use version solc <= 0.8.24 because the most recent versions of Hedera do not support all the opcodes introduced in the latest releases.